content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._es.md

```diff
@@ -1,32 +1,137 @@
+</div>
+
+<div class="margin-top-2">
+  <h2 id="phone-tips" class="margin-0">
+      Con un teléfono, realice fotografías del anverso y reverso de su documento de identidad
+  </h2>
+</div>
+
+Si está utilizando una computadora para verificar su identidad, puede cambiar a un teléfono para realizar fotografías de su documento de identidad.
+
+### Consejos para obtener fotografías nítidas con un teléfono
+
+<div class="grid-row grid-gap">
+  <div class="tablet:grid-col">
+    <img alt="El frente del documento de identidad está en el centro del marco de la cámara y sobre un fondo de tono oscuro." src="{{ site.baseurl }}/assets/img/help/id-dos-and-donts/id-do-front.png" />
+  </div>
+  <div class="tablet:grid-col">
+    <img alt="La parte posterior del documento de identidad está en el centro del marco de la cámara y sobre un fondo de tono oscuro." src="{{ site.baseurl }}/assets/img/help/id-dos-and-donts/id-do-back.png" />
+  </div>
+</div>
+
+{%
+  include components/icon-list.html
+  items=page.do_list
+  size='md'
+  icon_color='green'
+  icon_shape='check_circle'
+%}
+
+### Aspectos habituales al realizar fotografías de un documento de identidad
+
+<div class="grid-row grid-gap margin-bottom-6">
+  <div class="tablet:grid-col">
+    {%
+      include components/image-example-incorrect.html
+      image-alt-text='El documento de identidad se coloca sobre un fondo de color blanco.'
+      image='/assets/img/help/id-dos-and-donts/id-dont-white-background.png'
+      bordered_image=true
+      do_dont_text="NO"
+      text='Utilizar un fondo blanco'
+    %}
+  </div>
+  <div class="tablet:grid-col">
+    {%
+      include components/image-example-incorrect.html
+      image-alt-text='El documento de identidad se coloca sobre un fondo texturizado.'
+      image='/assets/img/help/id-dos-and-donts/id-dont-textured-background.png'
+      bordered_image=true
+      do_dont_text="NO"
+      text='Utilizar fondos con texturas o dibujos'
+    %}
+  </div>
+</div>
+<div class="grid-row grid-gap">
+  <div class="tablet:grid-col">
+    {%
+      include components/image-example-incorrect.html
+      image-alt-text='Los bordes del documento de identidad se encuentran recortados.'
+      image='/assets/img/help/id-dos-and-donts/id-dont-cropped.png'
+      bordered_image=true
+      do_dont_text="NO"
+      text='Recortar los bordes de la identificación'
+    %}
+  </div>
+  <div class="tablet:grid-col">
+    {%
+      include components/image-example-incorrect.html
+      image-alt-text='El documento de identidad presenta reflejos y sombras.'
+      image='/assets/img/help/id-dos-and-donts/id-dont-shadow.png'
+      do_dont_text="EVITE"
+      text='Brillos y sombras'
+    %}
+  </div>
+</div>
+
+<div class="page-content__horizontal-rule"></div>
+## Si no tienes acceso a un teléfono con cámara, puedes subir un archivo o utilizar un escáner.
+* Sigue los [mismos consejos para hacer fotos](#phone-tips) de tu documento de identidad con una cámara
```

Comment:
Just realized we changed the English text slightly to say "phone" instead of "camera". The Spanish translation should be:

> Sigue los mismos consejos para tomar fotografías con un teléfono

And the hyperlinked text should include "consejos para tomar fotografías con un teléfono"

Reply:
I think you copied the French translation in your comment, but yes the commit looks good!